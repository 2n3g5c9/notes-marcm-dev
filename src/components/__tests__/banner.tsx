import React from 'react'
import renderer from 'react-test-renderer'

import { Banner } from '@components'

describe('Banner', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Banner />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
