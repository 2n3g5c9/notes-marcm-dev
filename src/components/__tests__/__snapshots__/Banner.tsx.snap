// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
<div
  className="relative bg-accent"
>
  <div
    className="max-w-7xl mx-auto py-3 pl-8 sm:px-8"
  >
    <div
      className="pr-16 sm:text-center sm:px-16"
    >
      <p
        className="font-medium text-tertiary text-sm sm:text-base"
      >
        <span
          className="md:hidden"
        >
          This is a default text just to fill the space.
        </span>
        <span
          className="hidden md:inline"
        >
          This is a default text just to fill the space.
        </span>
      </p>
    </div>
  </div>
</div>
`;
